### PR TITLE
Twitter API integration

### PR DESCRIPTION
**Pull request by tjukes**
Updates PR #39

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -23,3 +27,35 @@ namespace :generate do
...
+  require 'dotenv/tasks'
+  require 'twitter'
+
+  desc 'Load Twitter config parameters as environment variables from Dotenv'
+  task :dotenv => :get_env do
+    @TWITTER_ENV = Dotenv.load.select { |key, _| key.match(/TWITTER/) }
+  end
+
+  desc 'Pull Tweets'
+  task :pull_tweets => :dotenv do
```
Comment (cody-code-wy) [inline Rakefile]: This probably should have been done in multiple commits. But it's not bad.

Comment (tjukes) [inline Rakefile]: Multiple commits would have meant committing incomplete functionality -- there are two separate things happening (establish Twitter API connection, pull data), but one doesn't do much without the other.

File: Rakefile [hunk 2]
```diff
@@ -11,10 +11,14 @@ namespace :db do
...
   require_relative "config/environment"
+end
+
+namespace :generate do
   desc 'Generates numbered tweets for every TwitterHandle in the database'
-  task :tweets do
+  task :tweets => :get_env do
     Tweet.destroy_all
     TwitterHandle.all.each do |handle|
       12.times do |number|
```
Comment (monicamow) [inline Rakefile]: what is this for? are we still testing on @cody-code-wy's dozen fake tweets? @tjukes

File: Gemfile [hunk 3]
```diff
@@ -20,6 +23,6 @@ end
 
 # bundle install --without test --without development
 group :production do
-  # use postgres in production, or move outside a group if your app uses postgres for development and production 
+  # use postgres in production, or move outside a group if your app uses postgres for development and production
```
Comment (monicamow) [inline Gemfile]: let me know if we are going to convert to postgres or not @cody-code-wy @tjukes

File: Rakefile (hunk 2 above)
Comment (cody-code-wy) [inline Rakefile]: @monicamow We don't want to remove it, it could be useful during testing.

File: Gemfile (hunk 3 above)
Comment (cody-code-wy) [inline Gemfile]: @monicamow we will, but you won't need to do anything, exactly because of the production group.

**Pull request merged**